blocklist_de-feed/operations.py

```diff
@@ -0,0 +1,138 @@
+""" Copyright start
+  Copyright (C) 2008 - 2022 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end """
+
+from connectors.core.connector import get_logger, ConnectorError
+import requests
+from time import sleep
+import arrow
+
+try:
+    from integrations.crudhub import trigger_ingest_playbook
+except:
+    # ignore. lower FSR version
+    pass
+
+logger = get_logger('blocklist_de-feed')
+
+SERVICE_MAPPING = {
+    "All": "all",
+    "SSH": 'ssh',
+    "Mail": 'mail',
+    "Apache": 'apache',
+    "IMAP": 'imap',
+    "FTP": 'ftp',
+    "SIP": 'sip',
+    "VOIP": "voip",
+    "Bots": 'bots',
+    "IRC Bot": "ircbot",
+    "Strong IPs": 'strongips',
+    "Brute Force Login": 'bruteforcelogin'
+}
+MAX_ATTEMPTS = 3
+LAST_HOURS = 48
+
+
+def validate_response(response):
+    if response.ok:
+        result = response.text
+        if 'text/plain' in response.headers["Content-Type"] and 'error' not in result.lower():
+            return result
+    logger.exception('Fail To request API {0} response is : {1}'.
+                     format(str(response.url), str(response.content)))
+    raise ConnectorError(
+        'Fail To request API {0} response is : {1}'.format(str(response.url), str(response.content)))
+
+
+def make_request(config, url, parameters=None, method='GET'):
+    verify_ssl = config.get('verify_ssl')
+    logger.debug("url: {}".format(url))
+    attempt = 1
+    status_code = 404
+    api_response = None
+    while attempt <= MAX_ATTEMPTS:
```

Comment:
can we do only a single retry when called from health check? Else health check takes very long for incorrect URL